maths/polynomial_evaluation.py

```diff
@@ -0,0 +1,26 @@
+def evaluate_poly(poly, x):
+    """
+        Objective: Computes the polynomial function for a given value x. 
+                Returns that value. 
+        Input Prams: 
+            poly: tuple of numbers - value of cofficients
+            x: value for x in f(x)
+        Return: value of f(x)
+    """
```

Comment:
Please add a doctest:
```
    >>> evaluate_poly((0.0, 0.0, 5.0, 9.3, 7.0), 10)
    79800.0
```
Then makes sure it passes locally with: __python3 -m doctest -v maths/polynomial_evaluation.py__